ui/desktop/tests/integration/components/hidden-secret-test.js

```diff
@@ -10,8 +10,20 @@ module('Integration | Component | hidden-secret', function (hooks) {
 
   test('it renders with a secret', async function (assert) {
```

Comment:
Can we also assert the secrets display correctly when you reveal it?

Reply:
Ditto

Reply:
yes